### PR TITLE
fix: print css

### PR DESCRIPTION
**Pull request by SiaChenRen**

### CODE REVIEW

File: src/templates/covidAppreciationLetter/template.tsx [hunk 1]
```diff
@@ -76,6 +77,19 @@ const Container = styled.div`
   .template-footer p span {
     margin-right: 5px;
   }
+
+  @media print {
+    .template-container {
+      box-shadow: none;
+      margin: 0px;
```
Comment (simboonlong) [inline src/templates/covidAppreciationLetter/template.tsx:84]: think can omit `px`

File: src/templates/covidAppreciationLetter/template.tsx [hunk 2]
```diff
@@ -76,6 +77,19 @@ const Container = styled.div`
...
+      border: none;
+    }
+  }
+
+  @page {
+    size: auto;
+    margin: 0px;
```
Comment (simboonlong) [inline src/templates/covidAppreciationLetter/template.tsx:91]: is `size: auto` the default initial value? if so then not necessary to define?

Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx:91]: Yes the size: auto is the default initial value, so I will remove it and keep the margin:0

File: src/templates/covidAppreciationLetter/template.tsx (hunk 1 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx:84]: Okay sure.

Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx]: Done

File: src/templates/covidAppreciationLetter/template.tsx (hunk 2 above)
Comment (SiaChenRen) [reply src/templates/covidAppreciationLetter/template.tsx]: Done

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**